runtime-main-mirror.yml

```diff
@@ -1,5 +1,16 @@
-# Disable triggers as it should only be called using Maestro
-trigger: none
+resources:
+  repositories:
+  - repository: internal dotnet-runtime
+    type: git
+    name: internal/dotnet-runtime
+    ref: main
+    trigger:
+      branches:
+        include:
+        - main
+
+trigger:
+- main
```

Comment:
I think this can be removed. You only need to trigger from runtime

Reply:
this is pointing to the `ref: main` in the `resource`. I think that just adding resource is not enough.